### PR TITLE
Feature/proxie

### PR DESCRIPTION
**Pull request by smoliji**
Adds a proxie script that creates a proxy to a gcp service

### CODE REVIEW

File: src/lib/proxie.ts [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+            let status = '❌';
+            try {
+                await commandExists(cmd);
+                status = '√';
+            } catch (error) {}
+            console.log(` ${cmd} ${status}`);
+            return { status, cmd };
+        })
+    ))
+        .filter(({ status }) => status !== '√');
```
Comment (vlasy) [inline src/lib/proxie.ts]: Extracting this symbol would be slightly safer for future

File: src/lib/proxie.ts [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+        if (data.length) {
+            return data;
+        }
+        list = gcloud.computeInstancesList({ project: answers.project });
+        return list;
+    };
+    return {
+        type: 'autocomplete',
+        name: 'vminstance',
+        message: 'VM Instance (mongo...)',
```
Comment (vlasy) [inline src/lib/proxie.ts:99]: Why `mongo`?

**Review by vlasy — APPROVED** [review]

File: src/lib/proxie.ts (hunk 2 above)
Comment (smoliji) [reply src/lib/proxie.ts:99]: I dont get the question.. why not?

The hint should be for developers to proxy to a service. And in our cases, mongo instances are either in pods, or in an standalone vm instance.

I will be glad to change this to anything that makes more sense.

File: src/lib/proxie.ts (hunk 1 above)
Comment (smoliji) [reply src/lib/proxie.ts]: Refactored, used the log symbols.

**Pull request merged**